examples/counter/lib/counter_view_model.dart

```diff
@@ -1,3 +1,4 @@
+import 'package:empire/empire_property.dart';
 import 'package:empire/empire_view_model.dart';
 
 class CounterViewModel extends EmpireViewModel {
```

Comment:
Maybe the example should be updated to use an EmpireIntProperty?

Reply:
As you wish my lord